src/tests/Microsoft.PowerFx.Interpreter.Tests/Docs/InterpreterBase.json

```diff
@@ -0,0 +1,140 @@
+{
+  "MinVersion": "1.3.0-local",
```

Comment:
This is a sample file for the interpreter. 
But each host would have their own file, base on their engine's configuration.